managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedCursor.java

```diff
@@ -587,4 +587,11 @@ void asyncFindNewestMatching(FindPositionConstraint constraint, Predicate<Entry>
      */
     void setThrottleMarkDelete(double throttleMarkDelete);
 
+    /**
+     * Get {@link ManagedLedger} attached with cursor
+     * 
+     * @return ManagedLedger
+     */
+    ManagedLedger getManagedLedger();
```

Comment:
seems this interface change need add implementation methods for all child class, including that in tests.

```
managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedCursorTest.java:[2079,13] method does not override or implement a method from a supertype
```

Reply:
yes..fixed it.